stable_baselines3/common/policies.py

```diff
@@ -199,7 +199,7 @@ def load_from_vector(self, vector: np.ndarray) -> None:
 
         :param vector:
         """
-        th.nn.utils.vector_to_parameters(th.FloatTensor(vector, device=self.device), self.parameters())
+        th.nn.utils.vector_to_parameters(th.as_tensor(vector, dtype=th.float, device=self.device), self.parameters())
```

Comment:
Actually, I'm pretty sure that this method is wrongly type hinted. It should be 

```diff
- def load_from_vector(self, vector: np.ndarray) -> None:
+ def load_from_vector(self, vector: th.Tensor) -> None:
```

Consequently, I wonder if the following could work:

```python
th.nn.utils.vector_to_parameters(vector.to(self.device), self.parameters())

Reply:
> Actually, I'm pretty sure that this method is wrongly type hinted. It should be

it is meant for evolution strategies, so converting candidates (numpy arrays) to parameters of a policy.

In practice, with ARS, I might use the GPU for matrix multiplication too, so probably should be `Union[np.ndarray, th.Tensor]` (I also need to check what I'm doing with CEM).

Reply:
@qgallouedec sorry for the ping, but I would like to have this one merge soon as ARS is currently broken...

Reply:
no problem, approving as a temporary fix